examples/dao/test/dao-app-flow.js

```diff
@@ -0,0 +1,456 @@
+  let proposalLsig;
+
+  this.beforeAll(async function () {
+    runtime = new Runtime([
+      master, creator, proposer, voterA, voterB,
+      depositLsigAcc, daoFundLsigAcc, proposalLsigAcc
+    ]);
+
+    flags = {
+      sender: creator.account,
+      localInts: 9,
+      localBytes: 7,
+      globalInts: 4,
+      globalBytes: 2
+    };
+  });
+
+  const getGlobal = (key) => runtime.getGlobalState(applicationID, key);
+
+  // fetch latest account state
+  function syncAccounts () {
+    // [
+    //   creator, proposer, voterA, voterB,
+    //   depositLsigAcc, daoFundLsigAcc, proposalLsigAcc
+    // ].map((acc) => runtime.getAccount(acc.address));
+    creator = runtime.getAccount(creator.address);
+    proposer = runtime.getAccount(proposer.address);
+    voterA = runtime.getAccount(voterA.address);
+    voterB = runtime.getAccount(voterB.address);
+    depositLsigAcc = runtime.getAccount(depositLsigAcc.address);
+    daoFundLsigAcc = runtime.getAccount(daoFundLsigAcc.address);
+    proposalLsigAcc = runtime.getAccount(proposalLsigAcc.address);
+  }
+
+  // DAO App initialization parameters
+  const deposit = 15; // deposit required to make a proposal
+  const minSupport = 5; // minimum number of yes power votes to validate proposal
+  const minDuration = 1 * 60; // 1min (minimum voting time in number of seconds)
+  const maxDuration = 5 * 60; // 5min (maximum voting time in number of seconds)
+  const url = 'www.my-url.com';
+
+  it('DAO flow test', () => {
+    govTokenID = runtime.addAsset(
+      'gov-token', { creator: { ...creator.account, name: 'dao-creator' } });
+
+    const creationFlags = Object.assign({}, flags);
+    const daoAppArgs = [
+      `int:${deposit}`,
+      `int:${minSupport}`,
+      `int:${minDuration}`,
+      `int:${maxDuration}`,
+      `str:${url}`
+    ];
+
+    const approvalProgram = getProgram('dao-app-approval.py', { ARG_GOV_TOKEN: govTokenID });
+    const clearProgram = getProgram('dao-app-clear.py');
+
+    // create application
+    applicationID = runtime.addApp(
+      { ...creationFlags, appArgs: daoAppArgs }, {}, approvalProgram, clearProgram);
+
+    // setup lsig account
+    // Initialize issuer lsig with bond-app ID
+    const scInitParam = {
+      ARG_GOV_TOKEN: govTokenID,
+      ARG_DAO_APP_ID: applicationID
+    };
+    const depositLsigProg = getProgram('deposit-lsig.py', scInitParam);
+    depositLsig = runtime.getLogicSig(depositLsigProg, []);
+    depositLsigAcc = runtime.getAccount(depositLsig.address());
+
+    const daoFundLsigProg = getProgram('dao-fund-lsig.py', scInitParam);
+    daoFundLsig = runtime.getLogicSig(daoFundLsigProg, []);
+    daoFundLsigAcc = runtime.getAccount(daoFundLsig.address());
+
+    const proposalLsigProg = getProgram('proposal-lsig.py',
+      { ARG_OWNER: proposer.address, ARG_DAO_APP_ID: applicationID });
+    proposalLsig = runtime.getLogicSig(proposalLsigProg, []);
+    proposalLsigAcc = runtime.getAccount(proposalLsig.address());
```

Comment:
can we move this according to flow written above ? or maybe add a comment here why we are declaring it before

Reply:
It's a part of the setup (compiling & funding lsig's). Will add a comment